lib/git-manager.js

```diff
@@ -342,30 +340,26 @@ function parseRepo(repo) {
  */
 function getIssueEvents(repo, number) {
    debug("Getting events for issue #%s", number);
-   return rateLimit(github.issues.getEvents)(params({ number }, repo))
-                       .then(paginate);
+   return github.paginate(githubRest.issues.listEventsForRepo, params({ issue_number: number }, repo))
 }
 
 function getIssueComments(repo, number) {
    debug("Getting comments for issue #%s", number);
-   return rateLimit(github.issues.getComments)(params({ number }, repo))
-                       .then(paginate);
+   return github.paginate(githubRest.issues.listComments, (params({ issue_number: number }, repo)));
 }
 
 function getPullReviewComments(repo, number) {
    debug("Getting pull review comments for pull #%s", number);
-   return rateLimit(github.pullRequests.getComments)(params({ number }, repo))
```

Comment:
Looks like the new rate limiter (thottling) is a little simpler, but may just work fine. If we're not using our own anymore, we should probably delete the file.

Reply:
https://github.com/iFixit/pulldasher/pull/211/commits/3394d440e0b4e722f1389a605e4f2e1ca2026d01